compose.yaml

```diff
@@ -7,7 +7,6 @@ services:
       - sigmaChat
     env_file:
       - .env.production
-

```

Comment:
Plz remove this change :////

Reply:
Done :) :)

Reply:
I left this one in as an exercise for the reviewer btw